### DISCUSSION TITLE
Welcome to DDEV-Local Discussions!

### DISCUSSION THREAD

**Discussion by rfay** [ddev/ddev#2683] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ElijahLynn** [comment]
Testing out Discussion user experience based on your [tweet](https://twitter.com/randyfay/status/1336371663886114816) @rfay! I am curious if it provides a real-time chat experience, which doesn't _seem_ to be the case. Love the transparency of it, but if it isn't that much different than an Issue, I currently don't see the value that much. Well, the transparency value is _HUGE_, but... I want real-time features for fast communication cycles.

**Reply by ElijahLynn** [reply]
Oh, threading, that is useful!

**Reply by ElijahLynn** [reply]
Sorry this isn't super focused on DDEV, btw.

**Comment by rfay** [comment]
I like that issues can be converted to conversations.

**Reply by ElijahLynn** [reply]
Oh? I didn't know this, that is neato.

**Comment by gilbertsoft** [comment]
Hey, I'm Simon, one of the many DDEV Local contributors. Working with DDEV on a daily base especially with my TYPO3 projects but doing a lot of other stuff, well DDEV has almost no limits. My main focuses are making DDEV better for the TYPO3 community and Windows users.

**Comment by iammati** [comment]
Hej, as my name already introduces myself, I'm Mati and I love to code using DDEV. It makes lots of fun honestly rather than just docker as you can simply configure everything which would be a big pain without those yaml-files haha. Anyways, I'm also a TYPO3 developer and also into Laravel (<- building at the moment a CMS using DDEV) trying to get out the maximal page speed with that combination of nginx without caching the HTML at all first, just to see how much you could do with the correct DDEV-/webserver-setup!
